ga4gh/datamodel/reads.py

```diff
@@ -432,17 +433,31 @@ class SimulatedReadGroup(AbstractReadGroup):
     """
     def __init__(self, parentContainer, localId, randomSeed, numAlignments=2):
         super(SimulatedReadGroup, self).__init__(parentContainer, localId)
+        self._randomSeed = randomSeed
 
     def getReadAlignments(self, referenceId=None, start=None, end=None):
+        rng = random.Random(self._randomSeed)
+        if start is not None:
+            # Skip ahead to start id
+            for i in xrange(start):
```

Comment:
The problem is that the random seeds don't just increment. The state derived from seed 2 isn't the state you get if you seed with 1 and then generate a value.

What we could do is generate each read from an individual seed of (base seed + read number), re-seeding for every read.